recipes-debian/sysvinit/sysvinit_debian.bb

```diff
@@ -23,7 +22,11 @@ SRC_URI += "file://install.patch \
            file://rc \
            file://rcS \
            file://bootlogd.init \
-           file://01_bootlogd"
+           file://01_bootlogd \
+"
+
+SRC_URI[md5sum] = "6eda8a97b86e0a6f59dabbf25202aa6f"
+SRC_URI[sha256sum] = "60bbc8c1e1792056e23761d22960b30bb13eccc2cabff8c7310a01f4d5df1519"
```

Comment:
These checksums are for poky's `${SAVANNAH_GNU_MIRROR}/sysvinit/sysvinit-${PV}.tar.bz2`
We don't use it, so these checksums are not necessary.

Reply:
That true. I'll remove them.